ci/pipeline.yml

```diff
@@ -0,0 +1,282 @@
+        skip_ssl_verify: false
+      vars_files:
+      - ops.git/monitoring/probe_endpoints.yml
+  on_failure:
+    put: slack
+    params:
+      text: |
+        :x: $ATC_EXTERNAL_URL - FAILED testing of prometheus boshrelease
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+  on_success:
+    put: slack
+    params:
+      text: |
+        :white_check_mark: $ATC_EXTERNAL_URL - Successfully tested prometheus boshrelease
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+- name: test-it-cleanup
+  plan:
+  - aggregate:
+    - get: slack
+      passed:
+      - test-it
+      trigger: true
+    - get: deployment-dev
+      passed:
+      - test-it
+    - get: deploy-prometheus.git
+      passed:
+      - test-it
+  - task: delete-dev-deployment
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: concourse/bosh-cli
+      run:
+        path: sh
+        args:
+        - -exc
+        - bosh delete-deployment -n
+        dir: ""
+      inputs:
+      - name: deploy-prometheus.git
+        path: ""
+    params:
+      BOSH_CA_CERT: ((bosh_ca_cert))
+      BOSH_CLIENT: ((bosh_client))
+      BOSH_CLIENT_SECRET: ((bosh_client_secret))
+      BOSH_DEPLOYMENT: prometheus-dev
+      BOSH_ENVIRONMENT: ((bosh_target))
+      BOSH_TARGET: ((bosh_target))
+  on_failure:
+    put: slack
+    params:
+      text: |
+        :x: $ATC_EXTERNAL_URL - FAILED cleaning up dev BOSH director
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+- name: ship-it-prod
+  plan:
+  - aggregate:
+    - get: stemcell
+      passed:
+      - test-it
+    - get: prometheus-boshrelease.github-release
+      passed:
+      - test-it
+    - get: ops.git
+      passed:
+      - test-it
+    - get: prometheus-boshrelease.git
+      passed:
+      - test-it
+    - get: deploy-prometheus.git
+      passed:
+      - test-it
+  - task: get-dns-values
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: govau/cga-cf-bosh-cli
+      inputs:
+      - name: deploy-prometheus.git
+      outputs:
+      - name: dns-details
+      run:
+        path: ./deploy-prometheus.git/ci/scripts/generate-dns-values.sh
+    params:
+      DNS_VAR_FILE: ./dns-details/dns-vars.yml
+  - put: deployment
+    params:
+      manifest: prometheus-boshrelease.git/manifests/prometheus.yml
+      ops_files:
+      - prometheus-boshrelease.git/manifests/operators/monitor-bosh.yml
+      - prometheus-boshrelease.git/manifests/operators/alertmanager-web-external-url.yml
+      # - prometheus-boshrelease.git/manifests/operators/alertmanager-slack-receiver.yml
```

Comment:
Is this deliberately disabled?

Reply:
the level of slack notifications is too noisy right now.  i've deliverately disabled until we get time to review and tune the alerts coming out of prometheus/alertmanager